src/unitxt/metrics.py

```diff
@@ -224,24 +300,11 @@ def compute(self, references: List[List[str]], predictions: List[str]) -> dict:
         return self._metric.compute(predictions=formatted_predictions, references=formatted_references)
 
 
-class SingleReferenceInstanceMetric(InstanceMetric):
-    def _compute(self, references: List[str], prediction: str) -> dict:
-        assert len(references) == 1, f"Expected only one reference , but received: {references}"
-        result = self.compute(references[0], prediction)
-        result["score"] = result[self.main_score]
-        result["score_name"] = self.main_score
-        return result
-
-    @abstractmethod
-    def compute(self, reference, prediction: str) -> dict:
-        pass
-
-
-class Accuracy(InstanceMetric):
+class Accuracy(ReferenceBasedInstanceMetric):
     reduction_map = {"mean": ["accuracy"]}
     main_score = "accuracy"
 
-    def compute(self, references: List[str], prediction: str) -> dict:
+    def compute(self, references: List[Any], prediction: Any) -> dict:
```

Comment:
why were the type hints changed?

Reply:
This is the current behavior.  The processed predictions and references are not strings.